src/LuaInterface.cpp

```diff
@@ -592,20 +593,20 @@ void LuaInterface::renameVar(TVar* var)
     } else {
         newName.append(R"([")" + vars.back()->getNewName() + R"("])");
     }
-    QString addString = QString(newName + " = " + oldName);
-    luaL_loadstring(L, addString.toLatin1().data());
+    QString variableRenameCode = QString(newName + " = " + oldVariableDeleteCode);
+    luaL_loadstring(L, variableRenameCode.toUtf8().constData());
     int error = lua_pcall(L, 0, LUA_MULTRET, 0);
     if (error) {
-        QString emsg = lua_tostring(L, -1);
+        QString emsg = QString::fromUtf8(lua_tostring(L, -1));
         var->clearNewName();
         return;
     }
     //delete it
-    oldName.append(QString(" = nil"));
-    luaL_loadstring(L, oldName.toLatin1().data());
+    oldVariableDeleteCode.append(QString(" = nil"));
+    luaL_loadstring(L, oldVariableDeleteCode.toUtf8().constData());
```

Comment:
If we have no further usage of `oldVariableDeleteCode` then _I think_ these two lines can be combined:

    luaL_loadstring(L, oldVariableDeleteCode.append(QLatin1String(" = nil")).toUtf8().constData());